DESCRIPTION

```diff
@@ -56,6 +56,7 @@ Suggests:
     nlme,
     profvis,
     quantreg,
+    RColorBrewer,
```

Comment:
Is there a reason we need this? I thought we access the brewer scales through the {scales} package, which already depends on {RColorBrewer}.

Reply:
It's to avoid an R CMD check warning (since my example is using `RColorBrewer`)

Reply:
Ah, I see it now, yes, agreed.